openvdb_ax/openvdb_ax/version.h

```diff
@@ -0,0 +1,76 @@
+// Copyright Contributors to the OpenVDB Project
+// SPDX-License-Identifier: MPL-2.0
+
+/// @file openvdb_ax/version.h
+///
+/// @brief Library and file format version numbers
+///
+/// @details  Based off of the versioning system within openvdb. There is
+///   currently no AX namespace versioning which instead leverages openvdb's
+///   version namespace.
+///
```

Comment:
I feel that since this controls the API/ABI, there isn't a reason for a secondary version structure here.  Either OpenVDB AX should use its own namespace based on its own version; or use the core name space and version.  Using the core namespace means for all practical purposes it is locked to the core version numbers!

I'd also be against using separate namespace as that encourages the idea these can desynchronize.  And suggests we need to say for Version X of AX, versions Y-Z of core are compatible. Which sounds a crazy explosion of issues.   If possible, I'd want to instead say that each version of AX is only compatible directly with the corresponding Core version, and any other cross over would require ABI guarantees and void *.

Reply:
Removed this file completely. AX is now always coupled to/part of the vdb version namespace